functions/index.js

```diff
@@ -31,6 +31,7 @@ const EMAIL_FROM = `"UW Blueprint" <info@uwblueprint.org>`;
 const SOCIAL_HTML = {
     facebook: '<a href="https://www.facebook.com/uwblueprint">Facebook</a>',
     instagram: '<a href="https://www.instagram.com/uwblueprint">Instagram</a>',
+    linkedin: '<a href="https://www.linkedin.com/company/uw-blueprint">LinkedIn</a>',
     medium: '<a href="https://www.medium.com/@uwblueprint">Medium</a>'
```

Comment:
```suggestion
    medium: '<a href="https://medium.com/uw-blueprint">Medium</a>'
```
Links it to our organization medium instead of the UW Blueprint author